.github/workflows/build.yml

```diff
@@ -10,7 +10,9 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-22.04
+          # renovate datasource: github-runners, depName: ubuntu
+          - ubuntu-20.04
```

Comment:
I intend on provoking a version update, to test the newly-updated `renovate.json`.